.repo-metadata.json

```diff
@@ -4,12 +4,13 @@
     "product_documentation": "https://cloud.google.com/logging/docs/reference/v2/rpc/google.appengine.logging.v1",
     "client_documentation": "https://cloud.google.com/python/docs/reference/appenginelogging/latest",
     "issue_tracker": "https://github.com/googleapis/python-appenginelogging/issues",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "OTHER",
     "repo": "googleapis/python-appengine-logging",
     "distribution_name": "google-cloud-appengine-logging",
     "api_id": "",
     "default_version": "v1",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "appenginelogging"
```

Comment:
This is not a GAPIC library, is `api_shortname` required?

Reply:
Fixed.